src/SystemServicesContext/useLocationPermissions.ts

```diff
@@ -41,6 +42,20 @@ const useLocationPermissions = (): LocationPermissions => {
     determineLocationEnabledState()
   }, [determineIsLocationRequired, determineLocationEnabledState])
 
+  useEffect(() => {
+    const subscription = subscribeToLocationStatusEvents((enabled: boolean) => {
+      if (enabled) {
+        setLocationEnabledState("On")
+      } else {
+        setLocationEnabledState("Off")
+      }
+    })
+
+    return () => {
+      subscription?.remove()
```

Comment:
why do we need the nullish operator here?

Reply:
I copied it from another subscription, I'll remove it